Service/NotificationManager.php

```diff
@@ -0,0 +1,93 @@
+<?php
+  /**
+   * Created by groovili
+   * https://github.com/groovili
+   */
+  declare(strict_types = 1);
+
+  namespace Kopay\NotificationBundle\Service;
+
+  use Kopay\NotificationBundle\Entity\Notification;
+  use Kopay\NotificationBundle\Entity\NotificationEmail;
+  use Kopay\NotificationBundle\Entity\NotificationMessageInterface;
+  use Kopay\NotificationBundle\Entity\NotificationPush;
+  use Symfony\Component\OptionsResolver\Exception\InvalidArgumentException;
+
+  /**
+   * Class NotificationManager
+   *
+   * @package Kopay\NotificationBundle\Service
+   */
+  class NotificationManager {
+    /**
+     * @var array
+     */
+    private $configuration;
+
+    protected const NOTIFICATION_TYPE = [
+        'Push'  => 0,
+        'Email' => 1,
+    ];
+
+    /**
+     * NotificationManager constructor.
+     *
+     * @param array $config
+     */
+    public function __construct(array $config)
+    {
+      $this->configuration = $config;
+    }
+
+    /**
+     * @param int $type
+     * @param array $options
+     */
+    public function create(int $type, array $options): ?NotificationMessageInterface
+    {
+      if(!isset($options['recipients']) || empty($options['recipients'])){
+        throw new InvalidArgumentException('Option "recipients" is 
+        required to create notification.');
+      }
+
+      if(!in_array($type, self::NOTIFICATION_TYPE)){
+        throw new InvalidArgumentException('Unknown type of notification.');
+      }
+
+      $title = (isset($options['title'])) ? $options['title'] : '';
+      $message = (isset($options['message'])) ? $options['message'] : '';
+
+      switch ($type) {
+        case self::NOTIFICATION_TYPE['Push']:
```

Comment:
Why I need to type 'Push'? 
I suggest 
const NOTIFICATION_PUSH
const NOTIFICATION_EMAIL